src/Raven.Server/Web/System/SetupHandler.cs

```diff
@@ -126,6 +128,14 @@ public async Task UserDomains()
                 var json = await context.ReadForMemoryAsync(RequestBodyStream(), "license activation");
                 var licenseInfo = JsonDeserializationServer.LicenseInfo(json);
 
+                var licenseStatus = LicenseManager.GetLicenseStatus(licenseInfo.License);
+                if (licenseStatus.Version.Major < 6)
+                {
+                    throw new LicenseLimitException(
+                        $"Your license ('{licenseStatus.Id}') version '{licenseStatus.Version}' doesn't allow you to use server version '{RavenVersionAttribute.Instance.FullVersion}'. " +
+                        $"Please proceed to the https://ravendb.net/l/8O2YU1 website to perform the license upgrade first.");
```

Comment:
This needs to be a UI element in the error.
Better yet, to get the error to be a proper one, not just generic error